testing/integration-tests/src/lib.rs

```diff
@@ -4,24 +4,27 @@
 
 #![deny(unused_crate_dependencies)]
 
-#[cfg(test)]
+#[cfg(all(test, not(feature = "unstable-light-client")))]
 mod codegen;
 #[cfg(test)]
 mod utils;
 
-#[cfg(test)]
+#[cfg(all(test, not(feature = "unstable-light-client")))]
```

Comment:
you could add another module called `full_client` or something to reduce the feature compilation noise

Reply:
Yep that makes a lot of sense, thanks!